test/unit_test/model/test_pytorch_model.py

```diff
@@ -63,7 +63,6 @@ def test_mlflow_model_hfconfig_function(self):
             },
         )
         # load_hf_model only works for huggingface models and not for mlflow models
-        assert mlflow_olive_model.get_hf_model_config() == hf_model.get_hf_model_config()
```

Comment:
is there a reason this test is removed?

Reply:
Yes, the hf_config in mlflow is slightly different with one got from huggingface.

Reply:
like the `_name_or_path`, `data_type` and etc, mlflow transformer model may overwrite some configs.